### ISSUE TITLE
Sampled particles for afterburner input is not a link

### ISSUE THREAD

**Issue by RenHirayama** [smash-transport/smash-vhlle-hybrid#42]
When the handler wants to use the output of a module as input for the other, we create a link, e.g. *Hydro/SMASH_IC.dat* is linked from *IC/[filename]* regardless of the filename. This doesn't seem to be the case between the *Sampler/particle_lists.oscar* and *Afterburner/sampled_particle_list.oscar*. Is this on purpose?

**Comment by AxelKrypton** [comment]
I do not think it was on purpose and I bet we just oversaw this. Do you want to fix this yourself or would you like me doing so? This should be a simple change to mention in the CHANGELOG, though.

**Comment by RobinSattler** [comment]
The current implementation looks intentional to me since the handler creates a new list of sampled particles and spectators if the `Add_spectators` key is set to true. A symbolic link is therefore not possible. If the key is set to `false`, then a symbolic link is created. 

For more detailed insights see function [`__static__Create_Sampled_Particles_List_File_Or_Symbolic_Link_With_Or_Without_Spectators()` in _bash/Afterburner_functionality.bash_](https://github.com/smash-transport/smash-vhlle-hybrid/blob/main/bash/Afterburner_functionality.bash#L80).

**Comment by AxelKrypton** [comment]
Thanks @RobinSattler for the remark. I am sure I wrote my comment without checking the code... and I forgot about the spectator different cases. Indeed, what you write looks correct. I mentioned the CHANGELOG file although I am aware we do not have one (yet), because I think it might be worth introducing one. Issue #37 exists already for that purpose.

**Comment by RobinSattler** [comment]
@AxelKrypton, I only realized this because I played around with the `Add_spectators` key looking for problems in the _add_spectators.py_ file (see issue #65) and stumbled upon this issue while looking for an already opened issue regarding the python file. 😄 I agree, a CHANGELOG would be nice to have.
@RenHirayama, I think this issue can be closed?

**Comment by AxelKrypton** [comment]
> @RenHirayama, I think this issue can be closed?

Yes, I think so. The implemented behaviour is intentional and it does what we want. There is nothing to be done here ATM.
